src/main/frontend/util/marker.cljs

```diff
@@ -13,14 +13,22 @@
 (defn add-or-update-marker
   [content format marker]
   (let [[re-pattern new-line-re-pattern]
-        (if (= :org format)
-          [#"\*+\s" #"\n\*+\s"]
-          [#"#+\s" #"\n#+\s"])
+        (case format
```

Comment:
`if` should work fine here. if not `:org`, then `:markdown` or `:md`.

Reply:
Fixed 😁.
Since Logseq is not markdown-first or org-mode-first, better make it explicit when searching keywords across source code.